pkg/apiserver/archive/archive.go

```diff
@@ -93,6 +93,11 @@ func (s *Service) list(c *gin.Context) {
 	name := c.Query("name")
 	ns := c.Query("namespace")
 
+	canList := utils.CanListChaos(c, ns)
+	if !canList {
+		return
```

Comment:
Is it more reasonable to return some errors here, such as returning error codes (403 and so on?) ?

Reply:
yes, you are right, I will refine this logic, maybe use the middleware @cwen0 mentioned